src/test/java/util/SupportUtils.java

```diff
@@ -128,6 +140,26 @@ public static boolean processCsvAndDump(byte[] bytes) throws IOException {
         return processor.parseRecords();
     }
 
+    /**
+     * Given a byte[] bytes, attempt to parse those bytes as CSV, transforming each record into
+     * JSON, separately posting that JSON to postUri.
+     * @param bytes
+     * @param postUri
+     * @return
+     * @throws IOException
+     */
+    public static boolean processCsvAndPost(byte[] bytes, String postUri) throws IOException {
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            String fakeRequestId = "" + System.currentTimeMillis();
+            System.out.println("About to process CSV and assign requestId: " + fakeRequestId);
```

Comment:
Instead of using the variable above (as it is only appending an empty string), would it be better to inline it here without the empty string?